#
# takeoff
#

takeoff_height: 1
takeoff_threshold: .25


#
# Follow Parameters
#
follow_x: 2.5
follow_x_kp: .3
follow_x_kd: .2
follow_x_ki: 0

follow_y: 0
follow_y_kp: .3
follow_y_kd: .2
follow_y_ki: 0

follow_z: -1
follow_z_kp: .2
follow_z_kd: 0
follow_z_ki: 0

follow_w: 0
follow_w_kp: 0
follow_w_kd: 0
follow_w_ki: 0

#
# Land Parameters
#

#These parameters specify the center of the landing area
land_x: 1.06
land_x_kp: .3
land_x_kd: .2
land_x_ki: 0

land_y: 0
land_y_kp: .3
land_y_kd: .2
land_y_ki: 0

land_z: .1 # this should be .1-.2m over the tag to allow the uav to maneuver
land_z_kp: .2
land_z_kd: 0
land_z_ki: 0

land_w: 0
land_w_kp: 0
land_w_kd: 0
land_w_ki: 0

land_height_threshold: .1
land_vel_mag: .1
land_radius: .1
land_angle_thresh: .13 # pi/24

#
# Return Parameters
#
return_gotoHeight: 5
return_finalHeight: 3
return_downRadius: 1
return_settleRadius: .5
return_tagTime: 1
return_tagLossThresh: 1
